app/main/posts/views/post-filters.service.js

```diff
@@ -103,6 +103,15 @@ function PostFiltersService(_, FormEndpoint, TagEndpoint, $q) {
         var query = _.omit(
             filters,
             function (value, key, object) {
+                if (key === 'saved_search') {
```

Comment:
just omitting the saved_search, reactiveFilters and qEnabled since we don't search for those in the Back